src/authorizer.ts

```diff
@@ -18,9 +21,10 @@ const logger = (msg: string, obj: any = {}) => {
   console.log(JSON.stringify({ "@msg": msg, ...obj }));
 };
 
-const deleteInstallation = async (payload: any) => {
+const deleteInstallation = async (payload: any, getPrivateKeyOptions: GetPrivateKeyOptions) => {
   const installationId = payload.installation.id;
-  const privateKey = process.env.PRIVATE_KEY;
+  const privateKey = getPrivateKey(getPrivateKeyOptions);
+  console.log(privateKey);
```

Comment:
```suggestion
```

Was this intentional? Logging the private key contents doesn't seem like a good security practice.

Reply:
Indeed it was not. Fixed.